<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>481</width>
     <height>320</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="pageSet">
    <widget class="QLabel" name="label_title">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>猜数字游戏</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>230</y>
       <width>321</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_start">
        <property name="text">
         <string>进入游戏</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>88</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_end">
        <property name="text">
         <string>退出游戏</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>130</y>
       <width>231</width>
       <height>40</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_guess">
        <property name="text">
         <string>          请设置猜数字时间(秒)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>20</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>40</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>60</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>80</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>100</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="pageGame">
    <widget class="QPushButton" name="pushButton_num0">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>249</y>
       <width>75</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_num1">
     <property name="geometry">
      <rect>
       <x>82</x>
       <y>249</y>
       <width>75</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_num2">
     <property name="geometry">
      <rect>
       <x>163</x>
       <y>249</y>
       <width>75</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_num3">
     <property name="geometry">
      <rect>
       <x>244</x>
       <y>249</y>
       <width>75</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_num4">
     <property name="geometry">
      <rect>
       <x>325</x>
       <y>249</y>
       <width>75</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_num5">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>283</y>
       <width>75</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_num6">
     <property name="geometry">
      <rect>
       <x>82</x>
       <y>283</y>
       <width>75</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_num7">
     <property name="geometry">
      <rect>
       <x>163</x>
       <y>283</y>
       <width>75</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_num8">
     <property name="geometry">
      <rect>
       <x>244</x>
       <y>283</y>
       <width>75</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_num9">
     <property name="geometry">
      <rect>
       <x>325</x>
       <y>283</y>
       <width>75</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>9</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_numComplete">
     <property name="geometry">
      <rect>
       <x>406</x>
       <y>249</y>
       <width>75</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>✔</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_numDelete">
     <property name="geometry">
      <rect>
       <x>406</x>
       <y>283</y>
       <width>75</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>×</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>400</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_process">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>50</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>时间进度</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_inputNum">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>输入1-10000数字:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_numTip">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>220</y>
       <width>75</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>提示</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_inputNum">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>5</number>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_back">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>0</y>
       <width>75</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>返回主界面</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pageWin">
    <widget class="QLabel" name="label_win">
     <property name="geometry">
      <rect>
       <x>119</x>
       <y>90</y>
       <width>251</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>恭喜你，猜对啦!</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pageEnd">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QWidget" name="widget" native="true">
       <widget class="QLabel" name="label_end">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>110</y>
          <width>251</width>
          <height>141</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>153</x>
          <y>60</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>很遗憾,时间结束!</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
